packages/forms-web-app/src/pages/have-your-say-guide/registering/config.js

```diff
@@ -4,6 +4,9 @@ const registeringRoute = 'registering-have-your-say';
 
 const registeringURL = getHaveYourSayGuidePageURL(registeringRoute);
 
+const registeringI18nNamespace = 'registering';
```

Comment:
I wonder if this is not a unique enough name for a namespace, and could potentially open us up for a naming collision in the future?

Reply:
similarly with packages/forms-web-app/src/pages/have-your-say-guide/get-involved/config.js above